### PR TITLE
Proofread Credential Offer Endpoint section

### PR DESCRIPTION
**Pull request by selfissued**
Correct issues identified while proofreading the Credential Offer Endpoint section.

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -302,18 +302,18 @@ For more details and concrete mitigations, see (#security_considerations_pre-aut
...
 The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client_metadata).
 
 The Credential Offer object, which is a JSON-encoded object with the Credential Offer parameters, can be sent by value or by reference.
 
-The Credential Offer contains a single URI query parameter `credential_offer` or `credential_offer_uri`:
+The Credential Offer contains a single URI query parameter, either `credential_offer` or `credential_offer_uri`:
 
-* `credential_offer`: Object with the Credential Offer parameters. This MUST NOT be present when `credential_offer_uri` parameter is present.
-* `credential_offer_uri`: String that is a URL using the `https` scheme referencing a resource containing a JSON-encoded object with the Credential Offer parameters. This MUST NOT be present when `credential_offer` parameter is present.
+* `credential_offer`: JSON object with the Credential Offer parameters. This MUST NOT be present when the `credential_offer_uri` parameter is present.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:315]: we agreed to use Object, not JSON object.

**Review by Sakurann — DISMISSED** [review]
Can we also keep the abbreviation "AS" for the authorization server? latest oauth wg specs/drafts (like security BCP) use "AS" without any issue.

**Review by paulbastian — APPROVED** [review]
looks good to me, only the point that Kristina already raised

**Comment by jogu** [comment]
> Can we also keep the abbreviation "AS" for the authorization server? latest oauth wg specs/drafts (like security BCP) use "AS" without any issue.

I'd personally suggest using "Authorization Server". I don't think we define the term "AS" anywhere and even if we did define it, I think it makes the specifications hard to understand for people new to the space.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (selfissued) [inline openid-4-verifiable-credential-issuance-1_0.md:315]:
```suggestion
* `credential_offer`: Object with the Credential Offer parameters. This MUST NOT be present when the `credential_offer_uri` parameter is present.
```

Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md]: I changed it back to Object.

**Review by bc-pi — APPROVED** [review]

**Review by peppelinux — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by awoie — APPROVED** [review]

**Pull request merged**